ajax/libs/cell/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "cell",
+  "filename": "cell.min.js",
+  "description": "A self-driving web app framework",
+  "homepage": "https://www.celljs.org",
+  "keywords": [],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/intercellular/cell.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "cell.js",
+          "cell.min.js"
+        ]
+      }
+    ]
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/intercellular/cell"
+  },
+  "license": "MIT"
```

Comment:
look dood i know you want to "Make the web a better place, one line at a time" but I don't think these lines are worth it. Try something where the order actually matters 🤔